src/rust/dbv2/migrations/2024-01-07-231321_tvl/up.sql

```diff
@@ -0,0 +1,99 @@
+-- Your SQL goes here
+CREATE MATERIALIZED VIEW aggregator.tvl_per_market AS
+SELECT
+    SUM(base_total) AS base_value,
+    SUM(quote_total) AS quote_value,
+    markets.market_id,
+    base_account_address, base_module_name, base_struct_name,
+    base_name_generic,
+    quote_account_address, quote_module_name, quote_struct_name
```

Comment:
Can we include in here the coin info from the coins endpoint? That will be especially useful for DeFi screener apps

An updated PR description would then be in order too